deploy_partition.yaml

```diff
@@ -10,9 +10,9 @@
 
     - name: check vagrant version
       fail:
-        msg: "Vagrant version of the host system ({{ lookup('file', playbook_dir + '/.vagrant_version_host_system') }}) does not match vagrant version in the deployment image ({{ result.stdout.split(' ')[1] }})"
+        msg: "Vagrant version of the host system ({{ lookup('file', '/mini-lab/.vagrant_version_host_system') }}) does not match vagrant version in the deployment image ({{ result.stdout.split(' ')[1] }})"
```

Comment:
I know this did not come from you... using absolute paths should be last resort. Should now be possible to revert back to `playbook_dir` since we didn't move the playbooks into a separate folder, right?